stdlib/grpc/src/main/resources/templates/skeleton/stub.mustache

```diff
@@ -18,7 +18,12 @@ public type {{serviceName}}{{#equals stubType "blocking"}}Blocking{{/equals}}Cli
 {{#blockingFunctions}}
     public remote function {{methodName}}({{#if inputType}}{{inputType}} req, {{/if}}grpc:Headers? headers = ()) returns ({{#if outputType}}[{{outputType}}, grpc:Headers]{{else}}grpc:Headers{{/if}}|grpc:Error) {
         {{#unless inputType}}Empty req = {};{{/unless}}
-        var payload = check self.grpcClient->blockingExecute("{{methodId}}", req, headers);
+        if !(self.grpcClient is grpc:Client) {
+            error err = error("UninitializedFieldsErrorType", message = "Field(s) are not initialized");
```

Comment:
This does not seem consistent with the normal error reasons we have, can we use something like `UninitializedClientError`

Reply:
However, I believe the change here should be to the init method itself. 

@daneshk can't we change the init method as 
```ballerina
        checkpanic c.initStub(self, "{{stubType}}", ROOT_DESCRIPTOR, getDescriptorMap());
        self.grpcClient = c;
```

Or at least move `self.grpcClient = c;` out of the else to make it always initialized?

Reply:
yes, we can